src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -13,13 +13,13 @@ if [ ! -d "$__PROJECT_DIR" ]; then
    exit 1
 fi
 
-if [ ! -d "$__TOOLS_DIR" ]; then
-   echo "ERROR: Cannot find tools path at '$__TOOLS_DIR'. Please pass in the tools directory as the 2nd parameter."
+if [ ! -e "$__DOTNET_CMD" ]; then
+   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 2nd parameter."
    exit 1
 fi
 
-if [ ! -e "$__DOTNET_CMD" ]; then
-   echo "ERROR: Cannot find dotnet.exe at path '$__DOTNET_CMD'. Please pass in the path to dotnet.exe as the 3rd parameter."
+if [ ! -d "$__TOOLRUNTIME_DIR" ]; then
```

Comment:
I wouldn't check if ToolRuntime dir doesn't exist as it is possible that this is the case, what I would check instead is:
```
if [ -z "$__TOOLRUNTIME_DIR" ]; then
```
which checks if the variable is defined, or alternatively you could also check if the variable is empty.